package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "arc.js-scripts",
+  "version": "0.0.0-alpha.0",
+  "description": "scripts that run against @daostack/arc.js",
+  "scripts": {
+    "clean": "rimraf build/**/*",
+    "build": "npm run clean && tsc",
+    "help": "node ./build/scripts/arcScript.js -h",
+    "script": "node ./build/scripts/arcScript.js",
+    "ganache": "npm explore @daostack/arc.js -- npm start ganache",
+    "migrateContracts": "npm explore @daostack/arc.js -- npm start migrateContracts",
+    "lint": "tslint -c tslint.json -e node_modules -p ./",
+    "lint.andFix": "npm run lint -- --fix"
+  },
+  "bin": {
+    "arcScript": "./scripts/arcScript.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dkent600/arc.js-scripts.git"
+  },
+  "keywords": [
+    "DAOstack",
+    "Arc.js",
+    "Arc",
+    "Ethereum",
+    "blockchain",
+    "DAO",
+    "javascript",
+    "smart",
+    "contracts"
+  ],
+  "author": "DAOstack.io",
+  "license": "GPL-3.0",
+  "bugs": {
+    "url": "https://github.com/dkent600/arc.js-scripts/issues"
+  },
+  "homepage": "https://github.com/dkent600/arc.js-scripts#readme",
+  "files": [
+    "scripts/",
+    "custom_typings/",
+    "tsconfig.json"
+  ],
+  "dependencies": {
+    "@daostack/arc.js": "0.0.0-alpha.111",
```

Comment:
please use arc.js or migration package which deploy arc rc.8

Reply:
This is one of the things I was hoping to submit in a separate PR -- it is ready to go